contents/tutorials/validating-what-you-ship.md

```diff
@@ -0,0 +1,224 @@
+|--------|----------------|-------------------|-----------|
+| Activation | Track completion of key actions that indicate value | Whether users are experiencing value, which predicts retention and downstream metrics | [Activation and why you should care](/newsletter/wtf-is-activation)
+| Retention | Week-over-week return rate | Does this have lasting impact? | [Track retention] (/docs/product-analytics/retention) |
+| Revenue | Conversion to paid | Did this affect the business? | [See Stripe revenue in PostHog] (https://posthog.com/docs/cdp/sources/stripe) |
+
+**Guardrail metrics** tell you if you broke something:
+
+| Metric | What to watch | Warning threshold | Resources |
+|--------|---------------|-------------------|-----------|
+| Support volume | Tickets tagged "onboarding" | >10% increase | [See Zendesk data in PostHog](/docs/data-warehouse/sources/zendesk) |
+| Error rate | Errors in onboarding flow | Any increase | [Setup error tracking](/docs/error-tracking/start-here) |
+| Performance | Load time of onboarding pages | >200ms increase |  |
+
+
+[Setup alerts to monitor these metrics] (https://posthog.com/docs/alerts)
+
+
+### Step 3: Create an A/B test for controlled rollout
+
+In PostHog, go to **Experiments** → **New experiment**.
+
+Create an experiment that splits users into control and test groups (start with 10-20% in the test group). This creates a natural control group: users in the control group give you a baseline for comparison.
+
+![A/B testing setup](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/abtest_setup_912f3a7d44.png)
+
+
+### Step 4: Build your dashboard before you ship
+
+Create a new dashboard called "New Onboarding Flow Launch - {Date}". In it, include the following insights:
+
+**Onboarding funnel** (Funnel insight):
+- Step 1: `onboarding_started`
+- Step 2: `onboarding_step_completed` where `step_name = profile_setup`
+- Step 3: `onboarding_step_completed` where `step_name = first_project`
+- Step 4: `onboarding_completed`
+- Filter by feature flag `new-onboarding-flow = true`
+
+**Completion over time** (Trends insight):
+- Events: `onboarding_started`, `onboarding_completed`
+- Compare: Users with feature flag on vs. off
+
+**Activation rate** (Trends insight):
+- Event: Your key activation event (e.g., `first_project_created`, `first_invite_sent`)
+- Compare: Users with feature flag on vs. off
+- This is your primary success metric
+
+**Guardrails** (Trends insight):
+- Support tickets tagged "onboarding"
+- Errors in onboarding flow
+- Time to complete onboarding (should decrease, not increase)
+
+
+### Step 5: Set up alerts and qualitative checkpoints
+
+**Proactive alerts** (Workflows → Slack notification):
+- Positive: Alert when a user completes onboarding and activates
+- Negative: Alert when support tickets tagged "onboarding" spike
+
+[Send PostHog event data to Slack webhooks](/docs/cdp/destinations/slack)
+
+**Post-interaction survey** (PostHog Surveys):
```

Comment:
```suggestion
**Post-interaction survey** (Surveys):
```

We basicallly never say "PostHog _product name_" (I htink that's stil lthe case)